tests/Aspire.Hosting.Azure.Tests/AzureStorageExtensionsTests.cs

```diff
@@ -163,4 +163,21 @@ public async Task AddAzureStorage_RunAsEmulator_SetSkipApiVersionCheck()
 
         Assert.Contains("--skipApiVersionCheck", args);
     }
+
+    [Fact]
```

Comment:
We should also have tests that verify the `AzureBlobStorageContainerResource`'s connection string (in both emulator and without) and tests for `IResourceWithAzureFunctionsConfig` when it is implemented.

Reply:
> We should also have tests that verify the AzureBlobStorageContainerResource's connection string (in both emulator and without)

Done

> tests for IResourceWithAzureFunctionsConfig when it is implemented.

There are no changes to the existing implementations and no new implementations.